Assets/MixedRealityToolkit/_Core/Utilities/InputMappingAxisUtility.cs

```diff
@@ -0,0 +1,356 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities
+{
+    /// <summary>
+    /// Utility class for Unity's Input Manager Mappings.
+    /// Input from all types should be defined here for use 
+    /// though out the entire toolkit.
+    /// </summary>
+    public static class InputMappingAxisUtility
+    {
+        #region Mixed Reality Xbox and Motion Controller constants
+
+        public const string CONTROLLER_LEFT_STICK_HORIZONTAL = "CONTROLLER_LEFT_STICK_HORIZONTAL";
+        public const string CONTROLLER_LEFT_STICK_VERTICAL = "CONTROLLER_LEFT_STICK_VERTICAL";
+        public const string CONTROLLER_RIGHT_STICK_HORIZONTAL = "CONTROLLER_RIGHT_STICK_HORIZONTAL";
+        public const string CONTROLLER_RIGHT_STICK_VERTICAL = "CONTROLLER_RIGHT_STICK_VERTICAL";
+        public const string XBOX_DPAD_HORIZONTAL = "XBOX_DPAD_HORIZONTAL";
+        public const string XBOX_DPAD_VERTICAL = "XBOX_DPAD_VERTICAL";
```

Comment:
Let's not add the Xbox controller Mappings, we won't need them where we're going.

Remember, we're getting the input from the windows namespace directly via `Windows.Gaming`

Reply:
Fair enough, I'll remove the Xbox controller "stuff"